ethcore/src/miner/miner.rs

```diff
@@ -649,7 +663,12 @@ impl MinerService for Miner {
 				// Client should send message after commit to db and inserting to chain.
 				.expect("Expected in-chain blocks.");
 			let block = BlockView::new(&block);
-			block.transactions()
+			let txs = block.transactions();
+			// populate sender
+			for tx in &txs {
+				let _sender = tx.sender();
+			}
```

Comment:
Populating `sender` is quite expensive operation - the idea is to do it in parallel (note `par_iter()`) and without locking transaction queue, but in fact I haven't verified what kind of gains it gives us in practice.